common/api/src/main/java/com/dfsek/terra/api/block/BlockType.java

```diff
@@ -13,6 +13,9 @@
 
 /**
  * Represents a type of block
+ * <p>
+ * Important! You <b>need</b> to implement a proper equals() and hashCode() here in
+ * your platform implementation for surface decorations to show up
```

Comment:
should probably be a separate PR

Reply:
We can also just leave it out here. I'll remove it for now